pymetis/__init__.py

```diff
@@ -128,6 +130,9 @@ def part_graph(nparts, adjacency=None, xadj=None, adjncy=None,
         all the edges of the graph have the same weight (i.e., the graph is
         unweighted), then the adjwgt can be set to ``None``.
 
+    METIS runtime options can be specified by supplying an Options object in
+    the input.
```

Comment:
Please expose options in sphinx docs.

Reply:
It's not clear to me if we can associate a docstring with the Options object
for sphinx to pick up. I added some 'manual' notes to the sphinx docs.